Configuration/AlCa/python/autoCond.py

```diff
@@ -36,7 +36,7 @@
     # GlobalTag for Run3 data relvals (express GT) - 140X_dataRun3_Express_v1 but snapshot at 2024-01-20 12:00:00 (UTC)
     'run3_data_express'            :    '140X_dataRun3_Express_frozen_v1',
     # GlobalTag for Run3 data relvals (prompt GT) - 140X_dataRun3_Prompt_v1 but snapshot at 2024-01-20 12:00:00 (UTC)
-    'run3_data_prompt'             :    '140X_dataRun3_Prompt_frozen_v1',
+    'run3_data_prompt'             :    '140X_dataRun3_Prompt_Candidate_2024_05_29_08_51_00',
```

Comment:
looks like that (minimally) you want to update *also* `run1_data` (which means the `run2_data` key), see:

https://github.com/cms-sw/cmssw/blob/421db781b56f97099271739f210c2aefd66a5c65/Configuration/AlCa/python/autoCond.py#L97

Reply:
Thanks Marco! Yes I'll definitely review all the GTs that need to be updated with this tag!
I was waiting from feedback on the JIRA because I think I'll also need to add another IOV to the new tag I created to differentiate 2023 and 2024 case (different triggers request).